server/src/test/java/org/elasticsearch/common/RoundingTests.java

```diff
@@ -195,6 +195,16 @@ public void testTimeUnitRoundingDST() {
         assertThat(tzRounding_chg.round(time("2014-11-02T06:01:01", chg)), isDate(time("2014-11-02T06:00:00", chg), chg));
     }
 
+    public void testOffsetRounding() {
```

Comment:
Let's add another test that does negative offsets too?

Reply:
Sure!

Reply:
I pushed a test with negative offsets.